### PR TITLE
chore: Add Snowflake release blog

### PR DESCRIPTION
**Pull request by yevgenypats**
This is still WIP but ready for first review.

### CODE REVIEW

File: website/pages/blog/announcing-cloudquery-snowflake-destination.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+author: yevgenypats
+---
+
+import { BlogHeader } from "../../components/BlogHeader"
+
+<BlogHeader/>
+
+## Introduction
+
+Cloud Infrastructure data exploded over the last decade which created the need to store those in data-warehouses and datalakes for analysis and reporting enabled by cheap storage. Today I'm excited to announce the release of our new [Snowflake destination plugin](../docs/recipes/destinations/snowflake), which enables you to sync all supported CloudQuery [source plugins](../docs/plugins/sources) to a [Snowflake](https://www.sqlite.org/index.html) database.
```
Comment (disq) [inline website/pages/blog/announcing-cloudquery-snowflake-destination.md:16]:
```suggestion
Cloud Infrastructure data exploded over the last decade which created the need to store those in data-warehouses and datalakes for analysis and reporting enabled by cheap storage. Today I'm excited to announce the release of our new [Snowflake destination plugin](../docs/recipes/destinations/snowflake), which enables you to sync all supported CloudQuery [source plugins](../docs/plugins/sources) to a [Snowflake](https://www.snowflake.com/) database.
```

File: website/pages/blog/announcing-cloudquery-snowflake-destination.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+## Use Cases
+
+### Cloud Infrastructure datalake and data-warehouse
+
+Snowflake is already used for ["Security data lake"](https://www.snowflake.com/en/data-cloud/workloads/cybersecurity/). CloudQuery unlocks bringing all your cloud [infrastructure configuration](../docs/plugins/sources) to the same place consolidating all your cloud infrastructure data in one place and enabling you to gain new insights.
+
+### Historical data
+
+Maintaing historical data is a common use case for data warehouses. By using Snowflake as a destination you can now store all your cloud infrastructure data in a data warehouse for long term storage, analysis and investigation use-cases.
```
Comment (disq) [inline website/pages/blog/announcing-cloudquery-snowflake-destination.md:26]:
```suggestion
Maintaining historical data is a common use case for data warehouses. By using Snowflake as a destination you can now store all your cloud infrastructure data in a data warehouse for long term storage, analysis and investigation use-cases.
```

**Review by disq — CHANGES_REQUESTED** [review]
initial review

File: website/pages/blog/announcing-cloudquery-snowflake-destination.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+## Syncing data
+
+Syncing data to snowflake can be done as with any other destination plugin so checkout our [Quickstart](../docs/quickstart) and [Snowflake recipe](../docs/recipes/destinations/snowflake).
```
Comment (shimonp21) [inline website/pages/blog/announcing-cloudquery-snowflake-destination.md:30]: 'check out'

**Review by shimonp21 — APPROVED** [review]

File: website/pages/blog/announcing-cloudquery-snowflake-destination.md (hunk 1 above)
Comment (hermanschaaf) [inline website/pages/blog/announcing-cloudquery-snowflake-destination.md]:
```suggestion
Cloud infrastructure data exploded over the last decade. Some teams using CloudQuery now collect data on more than 50 million (!) of their organization's cloud resources on a daily basis. This explosion has created the need to store this data in data warehouses and data lakes for better scaleability, analysis and reporting. Today I'm excited to announce the release of our new [Snowflake destination plugin](../docs/recipes/destinations/snowflake), which enables you to achieve this by syncing all supported CloudQuery [source plugins](../docs/plugins/sources) directly to a [Snowflake](https://www.sqlite.org/index.html) database.
```

File: website/pages/blog/announcing-cloudquery-snowflake-destination.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+Snowflake is already used as a ["Security data lake"](https://www.snowflake.com/en/data-cloud/workloads/cybersecurity/). CloudQuery brings all your cloud [infrastructure configuration data](../docs/plugins/sources) to the same place consolidating all your security and infrastructure data in one place and enabling new insights.
+
+### Historical data
+
+Maintaining historical data is a common use case for data warehouses. By using Snowflake as a destination you can now store all your cloud infrastructure data in a data warehouse for long term storage, analysis and investigation use-cases.
+
+## Syncing data
+
+Syncing data to snowflake can be done as with any other destination plugin so check-out our [Quickstart](../docs/quickstart) and [Snowflake recipe](../docs/recipes/destinations/snowflake).
```
Comment (hermanschaaf) [inline website/pages/blog/announcing-cloudquery-snowflake-destination.md:30]:
```suggestion
Syncing data to snowflake can be done as with any other destination plugin, so check out our [Quickstart](../docs/quickstart) and [Snowflake recipe](../docs/recipes/destinations/snowflake).
```

File: website/pages/blog/announcing-cloudquery-snowflake-destination.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+
+There are two ways to sync data to Snowflake:
+
+1. Direct (easy but not recommended for production or large data sets): This is the default mode of operation where CQ plugin will stream the results directly to the Snowflake database. There is no additional setup needed apart from authentication to Snowflake.
```
Comment (hermanschaaf) [inline website/pages/blog/announcing-cloudquery-snowflake-destination.md:34]:
```suggestion
1. Direct (easy but not recommended for production or large data sets): This is the default mode of operation where the CQ plugin will stream the results directly to the Snowflake database. There is no additional setup needed apart from authentication to Snowflake.
```

File: website/pages/blog/announcing-cloudquery-snowflake-destination.md [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+
+2. Loading via CSV/JSON from a remote storage: This is the standard way of loading data into Snowflake, it is recommended for production and large data sets. This mode requires a remote storage (e.g. S3, GCS, Azure Blob Storage) and a Snowflake stage to be created. The CQ plugin will stream the results to the remote storage. You can then load those files via a cronjob or via SnowPipe. This method is still in the works and will be updated soon with a guide.
+
+## Summary
+
+This is the first Snowflake destination plugin release with more improvments coming, and I'd love to hear your use-cases and ideas on how you leverage it. Feel free to open an issue on [GitHub](https://github.com/cloudquery/cloudquery) or join our [Discord](https://cloudquery.io/discord) to share your ideas.
```
Comment (hermanschaaf) [inline website/pages/blog/announcing-cloudquery-snowflake-destination.md:40]:
```suggestion
This is the first Snowflake destination plugin release with more improvements coming, and I'd love to hear your use-cases and ideas on how to leverage it. Feel free to open an issue on [GitHub](https://github.com/cloudquery/cloudquery) or join our [Discord](https://cloudquery.io/discord) to share your ideas.
```

**Review by hermanschaaf — CHANGES_REQUESTED** [review]
Maybe a screenshot of querying some data from Snowflake would be nice?

File: website/pages/blog/announcing-cloudquery-snowflake-destination.md (hunk 1 above)
Comment (yevgenypats) [reply website/pages/blog/announcing-cloudquery-snowflake-destination.md]: Thanks! Comited locally and pushed due to conflicts.

File: website/pages/blog/announcing-cloudquery-snowflake-destination.md [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+
+## Use Cases
+
+### Cloud Infrastructure data-lake and data-warehouse
+
+Snowflake is already used as a ["Security data lake"](https://www.snowflake.com/en/data-cloud/workloads/cybersecurity/). CloudQuery brings all your cloud [infrastructure configuration data](../docs/plugins/sources) to the same place consolidating all your security and infrastructure data in one place and enabling new insights.
+
+### Historical data
+
+Maintaining historical data is a common use case for data warehouses. By using Snowflake as a destination you can now store all your cloud infrastructure data in a data warehouse for long term storage, analysis and investigation use-cases.
```
[the remark this answers is not in the document]
Comment (yevgenypats) [reply website/pages/blog/announcing-cloudquery-snowflake-destination.md]: Thanks! Fixed

File: website/pages/blog/announcing-cloudquery-snowflake-destination.md [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+
+## Introduction
+
+Cloud Infrastructure data exploded over the last decade which created the need to store those in data-warehouses and datalakes for analysis and reporting enabled by cheap storage. Today I'm excited to announce the release of our new [Snowflake destination plugin](../docs/recipes/destinations/snowflake), which enables you to sync all supported CloudQuery [source plugins](../docs/plugins/sources) to a [Snowflake](https://www.snowflake.com/) database.
+
+## Use Cases
+
+### Cloud Infrastructure data-lake and data-warehouse
+
+Snowflake is already used as a ["Security data lake"](https://www.snowflake.com/en/data-cloud/workloads/cybersecurity/). CloudQuery brings all your cloud [infrastructure configuration data](../docs/plugins/sources) to the same place consolidating all your security and infrastructure data in one place and enabling new insights.
```
[the remark this answers is not in the document]
Comment (yevgenypats) [reply website/pages/blog/announcing-cloudquery-snowflake-destination.md]: Thx! Fixed

File: website/pages/blog/announcing-cloudquery-snowflake-destination.md [hunk 9]
```diff
@@ -0,0 +1,40 @@
...
+
+<BlogHeader/>
+
+## Introduction
+
+Cloud infrastructure data exploded over the last decade. Some teams using CloudQuery open source now collect data on more than 50 million (!) of their organization's cloud resources on a daily basis. This explosion has created the need to store this data in data warehouses and data lakes for better scalability, analysis and reporting. Today I'm excited to announce the release of our new [Snowflake destination plugin](../docs/recipes/destinations/snowflake), which enables you to achieve this by syncing all supported CloudQuery [source plugins](../docs/plugins/sources) directly to a [Snowflake](https://www.snowflake.com/) database.
+
+## Use Cases
+
+### Cloud Infrastructure data-lake and data-warehouse
```
Comment (hermanschaaf) [inline website/pages/blog/announcing-cloudquery-snowflake-destination.md:20]:
```suggestion
### Cloud Infrastructure data lake and data warehouse
```

File: website/pages/blog/announcing-cloudquery-snowflake-destination.md [hunk 10]
```diff
@@ -0,0 +1,40 @@
...
+
+There are two ways to sync data to Snowflake:
+
+1. Direct (easy but not recommended for production or large data sets): This is the default mode of operation where the CQ plugin will stream the results directly to the Snowflake database. There is no additional setup needed apart from authentication to Snowflake.
+
+2. Loading via CSV/JSON from a remote storage: This is the standard way of loading data into Snowflake, it is recommended for production and large data sets. This mode requires a remote storage (e.g. S3, GCS, Azure Blob Storage) and a Snowflake stage to be created. The CQ plugin will stream the results to the remote storage. You can then load those files via a cronjob or via SnowPipe. This method is still in the works and will be updated soon with a guide.
+
+## Summary
+
+This is the first Snowflake destination plugin release with more improvements coming, and I'd love to hear your use-cases and ideas on how to leverage it. Feel free to open an issue on [GitHub](https://github.com/cloudquery/cloudquery) or join our [Discord](https://cloudquery.io/discord) to share your ideas.
```
Comment (hermanschaaf) [inline website/pages/blog/announcing-cloudquery-snowflake-destination.md:40]:
```suggestion
This is the first Snowflake destination plugin release. More improvements are coming, and I'd love to hear your use-cases and ideas on how to leverage it. Feel free to open an issue on [GitHub](https://github.com/cloudquery/cloudquery) or join our [Discord](https://cloudquery.io/discord) to share your ideas.
```

**Review by hermanschaaf — CHANGES_REQUESTED** [review]
Looks good, but I still think a screenshot would be great :)

**Review by disq — APPROVED** [review]

File: website/pages/blog/announcing-cloudquery-snowflake-destination.md [hunk 11]
```diff
@@ -0,0 +1,44 @@
...
+
+Syncing data to snowflake can be done as with any other destination plugin, so check out our [Quickstart](../docs/quickstart) and [Snowflake recipe](../docs/recipes/destinations/snowflake).
+
+There are two ways to sync data to Snowflake:
+
+1. Direct (easy but not recommended for production or large data sets): This is the default mode of operation where the CQ plugin will stream the results directly to the Snowflake database. There is no additional setup needed apart from authentication to Snowflake.
+
+2. Loading via CSV/JSON from a remote storage: This is the standard way of loading data into Snowflake, it is recommended for production and large data sets. This mode requires a remote storage (e.g. S3, GCS, Azure Blob Storage) and a Snowflake stage to be created. The CQ plugin will stream the results to the remote storage. You can then load those files via a cronjob or via SnowPipe. This method is still in the works and will be updated soon with a guide.
+
+Once data is synced you can start querying either with the native Snowflake interface, code or with any BI tool that supports Snowflake. Here is an the equivalent of cloud infrastructure Hello World query:
```
Comment (hermanschaaf) [inline website/pages/blog/announcing-cloudquery-snowflake-destination.md:38]:
```suggestion
Once data is synced you can start querying either with the native Snowflake interface, code or with any BI tool that supports Snowflake. Here is the "Hello World" of cloud infrastructure  queries:
```

**Review by hermanschaaf — APPROVED** [review]

**Pull request merged**